service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -243,11 +253,20 @@ private void closeGracefully(Effect close) {
     private static class StaticServiceLocator extends CircuitBreakingServiceLocator {
         private final URI uri;
 
-        StaticServiceLocator(CircuitBreakers circuitBreakers, URI uri) {
-            super(circuitBreakers);
+        StaticServiceLocator(CircuitBreakersPanel circuitBreakersPanel, URI uri) {
+            super(circuitBreakersPanel);
             this.uri = uri;
         }
 
+        /**
+         * @deprecated Use constructor accepting {@link CircuitBreakersPanel} instead
+         */
+        @Deprecated
+        StaticServiceLocator(com.lightbend.lagom.internal.client.CircuitBreakers circuitBreakers, URI uri) {
+            this(CircuitBreakersConverter.toJavadslCircuitBreakerInvoker(circuitBreakers), uri);
+        }
```

Comment:
This is a private class, so I don't think we need to keep a deprecated constructor, we can just get rid of it.